components/process.rst

```diff
@@ -350,6 +350,22 @@ Use :method:`Symfony\\Component\\Process\\Process::disableOutput` and
     Moreover, you could not pass a callback to the ``start()``, ``run()`` or ``mustRun()``
     methods or use ``setIdleTimeout()``.
 
+Finding the Executable PHP Binary
+---------------------------------
+
+This component also provides a utility class called
+:class:`Symfony\\Component\\Process\\PhpExecutableFinder` which returns the
+absolute path of the executable PHP binary available on your server. It looks
+for the PHP executable using environment variables like ``PHP_BINARY``,
```

Comment:
I would remove this sentence: it is inaccurate, and it's complex to explain accurately
let just describe what it does, not how